src/coord/src/catalog.rs

```diff
@@ -28,7 +28,9 @@ use expr::{ExprHumanizer, GlobalId, MirScalarExpr, OptimizedMirRelationExpr};
 use repr::{ColumnType, RelationDesc, ScalarType};
 use sql::ast::display::AstDisplay;
 use sql::ast::{Expr, Raw};
-use sql::catalog::{Catalog as SqlCatalog, CatalogError as SqlCatalogError};
+use sql::catalog::{
+    Catalog as SqlCatalog, CatalogError as SqlCatalogError, CatalogItem as SqlCatalogItem,
```

Comment:
So so minor, but you went to all this trouble, you might as well import `CatalogItemType as SqlCatalogItemType`!

Reply:
Will avoid a trip through CI and put this in my next PR.